<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitIHM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_extra">
        <property name="text">
         <string>git</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_extra"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_extra">
        <property name="text">
         <string>Ex√©cuter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_branch">
        <property name="text">
         <string>Branche courante : ###</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_branchSwitch">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_branch">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_branchMerge">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_branch">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_unstaged">
        <property name="title">
         <string>Unstaged</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget_unstaged">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="itemAlignment">
            <set>Qt::AlignLeading</set>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_action">
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_reset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_checkout">
           <property name="text">
            <string>Checkout --</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_gitk">
           <property name="text">
            <string>Gitk</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_tags">
           <property name="text">
            <string>Tags...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_conflict">
           <property name="text">
            <string>Conflits...</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_stash">
             <property name="text">
              <string>Stash</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_pop">
             <property name="text">
              <string>Pop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_distantAction">
           <property name="title">
            <string>Distant</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QComboBox" name="comboBox_remote"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_push">
              <property name="text">
               <string>Push</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_fetch">
              <property name="text">
               <string>Fetch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_rebase">
              <property name="text">
               <string>Rebase</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_staged">
        <property name="title">
         <string>Staged</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget_staged">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="itemAlignment">
            <set>Qt::AlignLeading</set>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_commit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_commit">
        <property name="text">
         <string>Commit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_amend">
        <property name="text">
         <string>amend</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QToolButton" name="toolButton_gitDir">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_gitDir">
        <property name="text">
         <string>###</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_autoRefresh">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_timerTime">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Temps de rafra√Æchissement automatique (en secondes)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_refresh">
        <property name="toolTip">
         <string>Rafra√Æchir (Ctrl+F5)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
